packages/trip-form/src/MetroModeSelector/index.tsx

```diff
@@ -202,25 +219,41 @@ function ModeButton({
               <SubSettingsPane
                 modeButton={modeButton}
                 onSettingUpdate={onSettingsUpdate}
-                showControls={disableHover}
+                showControls={disableHover ?? false}
                 onDismiss={() => {
                   setOpen(false);
                 }}
                 onDisableMode={disableModeButton}
               />
             </HoverInnerContainer>
           </HoverPanel>
-        )}
-      </FloatingPortal>
+        </FloatingFocusManager>
+      )}
     </>
   );
 }
-
 interface Props {
+  /**
+   * Switches mode selector into click rather than hover mode, for mobile use.
```

Comment:
Does `disableHover = false` work on a large tablet?

Reply:
That prop is no longer needed with https://github.com/ibi-group/otp-ui/pull/13, see that PR for the behavior on mobile.